pkg/tsdb/tsdbtest/tsdbtest.go

```diff
@@ -72,7 +72,7 @@ func SetUp(t testing.TB, v3ioConfig *config.V3ioConfig) func() {
 	return func() {
 		defer metricReporter.Stop()
 		// Don't delete the TSDB table if the test failed
-		if !t.Failed() {
+		if !t.Failed() && !t.Skipped() {
```

Comment:
Why? `Setup` method should not be invoked at all it if test has been marked as "skipped"
In order to skip table delete attempt when using `deleteAll` you have to provide different Setup/TearDown

Reply:
will do it in another pr - must merge this today